test/fixtures/ccl/dcc-series-wallet-info-mask-state.yaml

```diff
@@ -709,6 +850,154 @@
             visible: true
             faqAnchor: mask_rules
             titleText: Maskenpflicht
-            subtitleText: Sie sind nicht von der Maskenpflicht ausgenommen
-            longText: Von der Maskenpflicht sind alle Personen befreit, die innerhalb der letzten 3 Monate geimpft wurden oder genesen sind oder innerhalb der letzten 24 Stunden negativ getestet wurden.
-            badgeText: Maskenpflicht
\ No newline at end of file
+            badgeText: Maskenpflicht
+    # federal default rules
+    - time: rat1+P5D # some long duration
+      scenarioIdentifier: ''
+      assertions:
+        walletInfo:
+          maskState:
+            longText:
+              de: 'Sie sind nicht von der Maskenpflicht ausgenommen.\n\nVon der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen, die innerhalb der letzten 24 Stunden negativ getestet wurden.'
```

Comment:
Also repeated here:

"Von der Maskenpflicht sind alle Personen Von der Maskenpflicht sind alle Personen ausgenommen"

Reply:
Thanks @MikeMcC399 for spotting this, fixed with https://github.com/corona-warn-app/cwa-app-ccl/pull/106/commits/da8f1ab3bc1832f6987fc5abd9108d9cc5f98ac0